terraform/outputs.tf

```diff
@@ -0,0 +1,30 @@
+output "access_key" {
+  value       = module.user.access_key
+  description = "The IAM access key associated with the CI IAM user created by this module."
```

Comment:
What does CI stand for in this case? I looked in the README - we should define the acronym somewhere, ideally where it's first used.

Reply:
From usage elsewhere, it's likely the standard `continuous integration` and not something `certificate` related.

Reply:
Continuous Integration, per the skeletons. Resolving this since I was just confused by being in a repo about certs.